tests/test_hello.py

```diff
@@ -81,24 +81,35 @@ def test_hello_sdist():
     sdists_zip = glob.glob('dist/*.zip')
     assert sdists_tar or sdists_zip
 
-    expected_content = [
-        'hello-1.2.3',
-        'hello-1.2.3/CMakeLists.txt',
-        'hello-1.2.3/hello',
-        'hello-1.2.3/hello/_hello.cxx',
-        'hello-1.2.3/hello/CMakeLists.txt',
-        'hello-1.2.3/hello/__init__.py',
-        'hello-1.2.3/hello/__main__.py',
-        'hello-1.2.3/setup.py',
-        'hello-1.2.3/PKG-INFO'
-    ]
-
-    member_list = []
+    member_list = None
+    expected_content = None
     if sdists_tar:
+        expected_content = [
+            'hello-1.2.3',
+            'hello-1.2.3/CMakeLists.txt',
+            'hello-1.2.3/hello',
+            'hello-1.2.3/hello/_hello.cxx',
+            'hello-1.2.3/hello/CMakeLists.txt',
+            'hello-1.2.3/hello/__init__.py',
+            'hello-1.2.3/hello/__main__.py',
+            'hello-1.2.3/setup.py',
+            'hello-1.2.3/PKG-INFO'
+        ]
         member_list = tarfile.open('dist/hello-1.2.3.tar.gz').getnames()
+
     elif sdists_zip:
+        expected_content = [
+            'hello-1.2.3/CMakeLists.txt',
+            'hello-1.2.3/hello/_hello.cxx',
+            'hello-1.2.3/hello/CMakeLists.txt',
+            'hello-1.2.3/hello/__init__.py',
+            'hello-1.2.3/hello/__main__.py',
+            'hello-1.2.3/setup.py',
+            'hello-1.2.3/PKG-INFO'
+        ]
         member_list = ZipFile('dist/hello-1.2.3.zip').namelist()
```

Comment:
Once codecov passes, feel free to add it in.  The same simplification can be done in `test_hello_cython.py` and `test_hello_pure.py`.